indra/llwindow/llwindowwin32.cpp

```diff
@@ -422,13 +400,13 @@ struct LLWindowWin32::LLWindowWin32Thread : public LL::ThreadPool
 
     using FuncType = std::function<void()>;
     // call GetMessage() and pull enqueue messages for later processing
-    void gatherInput();
     HWND mWindowHandleThrd = NULL;
     HDC mhDCThrd = 0;
 
     // *HACK: Attempt to prevent startup crashes by deferring memory accounting
     // until after some graphics setup. See SL-20177. -Cosmic,2023-09-18
     bool mGLReady = false;
+    bool mGotGLBuffer = false;
     LLAtomicBool mDeleteOnExit = false;
```

Comment:
Everything from here to the end of the struct can go

Reply:
Right. Thank you, @Ansariel!